src/Login/components/LoginComponent.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Login = (
+    {
+        handleSubmit,
+        onChange,
+        email,
+        password,
+        errors
+    }
+) => {
+    return (
+
+        <div className="container">
+            <h1>Login</h1>
+            <div className="myform">
+                <form >
+                    <div className="form-input"  id="errormessage">
+                        {/* <label>Email:</label> */}
```

Comment:
Commented out code

Reply:
@AGMETEOR  kingly not that l worked on this